src/client/common/process/logger.ts

```diff
@@ -38,7 +38,7 @@ export class ProcessLogger implements IProcessLogger {
         }
 
         info.forEach((line) => {
-            traceInfo(line);
+            traceInfo(`Process Execution: ${line}`);
```

Comment:
It was difficult to tell where the message `cwd: <path` was coming from.
This helps us on CI figure out the source of such messages.